src/sign-in-wpf/MainWindow.xaml.cs

```diff
@@ -0,0 +1,122 @@
+﻿using Microsoft.Identity.Client;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using System.Windows;
+
+namespace MsalExample
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        private readonly HttpClient httpClient = new();
+
+        // Generally, your MSAL client will have a lifecycle that matches the lifecycle
+        // of the user's session in the application. In this sample, the lifecycle of the
+        // MSAL client to the lifecycle of this form.
+        private readonly IPublicClientApplication msalPublicClientApp;

```

Comment:
```suggestion
        private readonly IPublicClientApplication _msalPublicClientApp;

```

Per [C# code conventions](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#camel-case), here and any references.

Reply:
Shoot, can't believe I missed this. Thanks!

Reply:
Addressed in https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/26/commits/6b0aa75aca1f146453b6230951d38bef4b8ba050